### ISSUE TITLE
Support prettier-tslint

### ISSUE THREAD

**Issue by zimme** [prettier/prettier-atom#302]
I would like to request support for [`prettier-tslint`](https://github.com/azz/prettier-tslint).

**Comment by zimme** [comment]
However it seems to be blocked by the following issue.
https://github.com/azz/prettier-tslint/issues/1

**Comment by chibicode** [comment]
@zimme this issue is now closed https://github.com/azz/prettier-tslint/issues/1

**Comment by sabina-clover** [comment]
Any progress on this? Does atom now have support for prettier-tslint?

**Comment by j-f1** [comment]
Now that the TypeScript team is shifting their focus to ESLint (https://github.com/Microsoft/TypeScript/issues/29288), is this feature still needed?

**Comment by robwise** [comment]
@j-f1 Good to know (I'm totally out of touch with TS-related things).

Given that it seems the TS team is shifting towards ESLint (which we already support) and away from TSLint, yes, at first glance, maybe we can avoid having to implement this after all?

**Comment by zimme** [comment]
I've pretty much switched over to vscode so I'll unfollow this issue as it's no longer relevant for me.
